com.civicrm.hrjobroles/CRM/Hrjobroles/Upgrader.php

```diff
@@ -78,6 +78,25 @@ public function upgrade_1006() {
     return TRUE;
   }
 
+  /**
+   * Creates an option group for organisation contact
+   * used for job role funder
+   *
+   * @return bool
+   */
+  public function upgrade_1007() {
+    $result = civicrm_api3('OptionGroup', 'get', [
+      'name' => 'hrjc_funders',
```

Comment:
I think we should use singular here too to be consistent also with previous option groups for job contracts and job roles like: hrjc_department, hrjc_location e.t.c